$NetBSD: patch-aa,v 1.3 2012/09/01 11:18:54 marino Exp $

# Add support for NetBSD - copy of FreeBSD except for uname -m amd64 vs x86_64
# Add support for DragonFly

--- configure.orig	2010-09-14 06:00:09.000000000 +0000
+++ configure
@@ -89,7 +89,7 @@ build_config()
 		echo "RDYNAMIC=-rdynamic"
 		echo "LDFLAGS+=-Wl,--no-undefined -Wl,--as-needed"
 		;;
-		FreeBSD)
+		FreeBSD|DragonFly)
 		[ "$(uname -m)" = "x86_64" ] && echo "ARCH_X86_64=1" && echo "CFLAGS+=-DARCH_X86_64"
 		echo "OPTIMISATIONS+=-ffast-math"
 		echo "CFLAGS+=-pthread"
@@ -97,6 +97,14 @@ build_config()
 		echo "RDYNAMIC=-rdynamic"
 		echo "LDFLAGS+=-Wl,--no-undefined -Wl,--as-needed"
 		;;
+		NetBSD)
+		[ "$(uname -m)" = "amd64" ] && echo "ARCH_X86_64=1" && echo "CFLAGS+=-DARCH_X86_64"
+		echo "OPTIMISATIONS+=-ffast-math"
+		echo "CFLAGS+=-pthread"
+		echo "SHFLAGS=-shared"
+		echo "RDYNAMIC=-rdynamic"
+		echo "LDFLAGS+=-Wl,--no-undefined -Wl,--as-needed"
+		;;
 		*)
 		;;
 		esac
@@ -171,7 +179,7 @@ case $targetos in
 	Darwin)
 	LIBSUF=".dylib"
 	;;
-	Linux|FreeBSD)
+	Linux|FreeBSD|NetBSD|DragonFly)
 	LIBSUF=".so"
 	;;
 	*)
